src/MongoDB.Driver.Core/Core/Operations/FindOperation.cs

```diff
@@ -277,6 +277,7 @@ public BsonDocument Min
         /// <value>
         /// The additional query modifiers.
         /// </value>
+        [Obsolete("Use individual setters instead.")]
```

Comment:
Let's say "properties" instead of "setters":

    [Obsolete("Use individual properties instead.")]

It's the property that's being deprecated, not just the setter.